src/app/icd/ICDConfigurationData.h

```diff
@@ -97,15 +103,31 @@ class ICDConfigurationData
 
     static constexpr uint32_t kMinLitActiveModeThreshold_ms = 5000;
 
-    static_assert((CHIP_CONFIG_ICD_IDLE_MODE_DURATION_SEC) <= 64800,
+    /**
+     * @brief Change the ActiveModeDuration and IdleModeDuration value
+     *        To only change one value, pass the old value for the other one
```

Comment:
Suggest having Optional for the args, that way you don't need to read-modify-write with old one.

Nit: split sentences w/ periods.